web/src/user_group_pill.ts

```diff
@@ -72,7 +93,10 @@ function get_group_members(user_group: UserGroup): number[] {
     return user_ids.filter((user_id) => people.is_person_active(user_id));
 }
 
-export function append_user_group(group: UserGroup, pill_widget: CombinedPillContainer): void {
+export function append_user_group(
+    group: UserGroup,
+    pill_widget: CombinedPillContainer | UserGroupPillWidget,
+): void {
```

Comment:
`CombinedPillContainer` already contains the type `UserGroupPillWidget` (in the file `typeahead_helper.ts`, no need to mention here with `|` symbol. Other functions in this file can be changed similarly.

Reply:
it is `UserGroupPill` not `UserGroupPillWidget`
```ts
export type CombinedPill = StreamPill | UserGroupPill | UserPill;
export type CombinedPillContainer = InputPillContainer<CombinedPill>;
```

or I am missing something ?

Reply:
Yeah I think this type is OK.